app/models/stop_searcher.rb

```diff
@@ -2,18 +2,18 @@ class StopSearcher
   include ActiveModel::SerializerSupport
   DEFAULT_PER_PAGE = 20
 
+  attr_reader :per_page, :page
+
   def initialize(params)
     @params = params
+    @per_page = params.fetch(:per_page) { DEFAULT_PER_PAGE }.to_i
+    @page = params.fetch(:page) { 1 }.to_i
   end
```

Comment:
What's your reasoning for doing the block form of fetch instead of `params.fetch(:page, 1).to_i`? Just curious.